samples/proto/person_profile.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package samples.protobuf;
+option java_package = "samples.protobuf";
+
+import "corfu_options.proto";
+import "work_experience.proto";
+
+message Name {
+    string first_name = 1 [(org.corfudb.runtime.schema).secondary_key = true];
+    string last_name = 2 [(org.corfudb.runtime.schema).secondary_key = true];
+}
+
+message Person {
+    string first_name = 1 [(org.corfudb.runtime.schema).secondary_key = true];
```

Comment:
Why first_name and last name are not replaced by Name?

Reply:
Done.